README.md

```diff
@@ -5,17 +5,20 @@ charts-build-scripts
 
 ## Before running the scripts
 
-#### If you are creating a new charts repository with `charts-build-scripts init`
+#### If you are creating a new charts repository 
 
-Set the environment variable `GITHUB_AUTH_TOKEN` to your [personal Github Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).
+Checkout into the Git branch that corresponds to your Source, Staging, or Live branch.
```

Comment:
nit: "Checkout into the" -> "Checkout the"

Reply:
will update, good catch.